src/sentry/grouping/ingest/config.py

```diff
@@ -86,13 +86,6 @@ def is_in_transition(project: Project) -> bool:
 
 
 def project_uses_optimized_grouping(project: Project) -> bool:
-    primary_grouping_config = project.get_option("sentry:grouping_config")
-    secondary_grouping_config = project.get_option("sentry:secondary_grouping_config")
-    has_mobile_config = "mobile:2021-02-12" in [primary_grouping_config, secondary_grouping_config]
-
-    if has_mobile_config or options.get("grouping.config_transition.killswitch_enabled"):
-        return False
```

Comment:
Until we have a few days of being 100% on the optimized transition codepath, can we please keep the killswitch check here?